caps/capabilities_test.go

```diff
@@ -46,7 +46,7 @@ func (s *CapabilitySuite) TestValidateName(c *C) {
 
 func (s *CapabilitySuite) TestAdd(c *C) {
 	repo := NewRepository()
-	cap := &Capability{"name", "label", FileType}
+	cap := &Capability{Name: "name", Label: "label", Type: FileType}
```

Comment:
you'll probably be happier in the long run if you make this multi-line, but I don't really care at this point :-)